app/controllers/concerns/hyrax/works_controller_behavior.rb

```diff
@@ -101,7 +100,6 @@ def update
       else
         respond_to do |wants|
           wants.html do
-            curation_concern.errors[:single_collection] << curation_concern.errors.messages[:collections].to_sentence
```

Comment:
This is only doing a rename of the error message.  It is not needed. _form_collections_error.html.erb partial is what actually displays this message.